docs/server-backup-version-comparison.md

```diff
@@ -1,50 +1,38 @@
 
-!!! admonition "See also"
-   
-    [How XtraBackup works](how-xtrabackup-works.md)
+Here are the possible scenarios and the actions taken:
 
-## Version changes
+| Scenario                                                                                  | Actions Taken                                                                                                                                            |
+|----|----|
+| Server version is the same as Percona XtraBackup version                                  | The backup process continues without any issues. This is the ideal scenario because both the server and the backup tool are fully compatible.         |
+| Server version is older than Percona XtraBackup version                                  | The backup process continues without any issues. Percona XtraBackup is designed to handle older versions of the server without problems.             |
+| Server version is newer than Percona XtraBackup version                                  | The backup process stops with an error unless you explicitly override the version check. This happens because the backup tool may not support new features or changes introduced in the newer server version. |
+| Server version is newer than Percona XtraBackup version, and the version check is overridden | The backup process continues, but there is a risk of failure or corruption. Overriding the version check can be risky because the backup tool might not be able to handle new features or changes correctly. |
 
-With the release of Percona XtraBackup 8.0.34-29, Percona XtraBackup allows backups on version 8.0.35 and higher. 
+Until version 8.0.34, to ensure successful backups and restores, it is recommended to use a Percona XtraBackup version that matches or is newer than your server version. However, version 8.0.34 can back up newer server versions and older server versions if you set the `lock-ddl=ON` option.
 
-Percona XtraBackup 8.0.21 adds the `--no-server-version-check` option.
+## Version updates
 
-## Version check
+Percona XtraBackup 8.0.34: Disabled the server-version-check.
 
-Before the backup starts, XtraBackup compares the source system version to
-the Percona XtraBackup version. If the source system version is greater than the XtraBackup version, XtraBackup stops the backup and returns an
-error message. This comparison prevents a failed or corrupted
-backup due to source system changes.
+Percona XtraBackup 8.0.22-15: introduced `--no-server-version-check`. This option allows you to override the version comparison process. This option is off by default, meaning the comparison occurs unless explicitly overridden.
 
```

Comment:
Here:
Percona XtraBackup 8.0.34: `Disabled `

and here

Percona XtraBackup 8.0.22-15: `introduced `

Make them both start from lowercase